src/utilities/contentful/keys.js

```diff
@@ -0,0 +1,6 @@
+const keys = {
+  space: '0f39zonxf59w',
+  accessToken: '10OGNlSRGeKn81WAaTUxMjVL0nhXFEUszwRJIY7vPeI',
```

Comment:
Thanks @rebeccahongsf & @moisesnarvaez 

I've gone ahead and have put the items into vault. You can pull them down by running `npm run vault` on your local and they will be available on all future builds.  

```suggestion
  space: process.env.ASSOCIATES_CONTENTFUL_SPACE,
  accessToken: process.env.ASSOCIATES_CONTENTFUL_TOKEN,
```

Reply:
Thank you! I'll implement this